gb-cpu/src/memory/consts.rs

```diff
@@ -0,0 +1,17 @@
+pub const ROM_MIN: u16 = 0x000;
+pub const ROM_MAX: u16 = 0x7fff;
+//pub const VRAM_MIN: u16 = 0x8000;
```

Comment:
Why commenting those, if you will use them do not comment them or you may forget them (unlikely, but prefer keeping the warning to remember that you have those constant to use)

Reply:
It was exactly for the warnings and no way i forget to use thoses as they are mandatory for the Memory implementation. That's a detail really

Reply:
If you are not using them yet, I believe you can use the prefix `_` to silence the warning. You'll remove it when you will start using the variable.